src/js/components/search/topFilterBar/filterGroups/TopFilterGroupGenerator.jsx

```diff
@@ -40,6 +41,8 @@ export const topFilterGroupGenerator = (config = {
             return <AgencyFilterGroup key={groupKey} {...config} />;
         case 'selectedRecipients':
             return <RecipientFilterGroup key={groupKey} {...config} />;
+        case 'keyword':
```

Comment:
Same comment as in others - can we move this to the top to reflect the order of the filters in the Sidebar?